test/test_device_hydro_generation_constructors.jl

```diff
@@ -762,19 +767,23 @@ end
 
     for net in networks, mod in models, p in parameters_value
         @testset "$(mod) ED model on $(net) and use_parameters = $(p)" begin
-            devices = Dict{String, DeviceModel}(
-                :Generators => DeviceModel(ThermalStandard, ThermalDispatch),
-                :Loads => DeviceModel(PowerLoad, StaticPowerLoad),
-                :HydroGens => DeviceModel(HydroEnergyReservoir, mod),
-            )
-            template = OperationsProblemTemplate(net, devices, branches, services)
+            template = OperationsProblemTemplate(net)
```

Comment:
move this template definition to the templates folder. I am trying to reduce the number of templates out there in the tests and their modifications.

Reply:
I decided to use the existing template and just add the hydro device to it.